evm/x/namespace/keeper/keeper.go

```diff
@@ -14,6 +18,12 @@ type Keeper struct {
 }
 
 func NewKeeper(storeKey *storetypes.KVStoreKey, auth string) *Keeper {
+	if auth == "" {
+		auth = authtypes.NewModuleAddress(sequencer.Name).String()
+		if strings.Contains(auth, "cosmos") {
+			log.Fatal("address had 'cosmos' bech32 prefix, should be 'world'")
```

Comment:
Should this check be fore a "cosmos" prefix, or just that it contains cosmos? The error message and if check are not in agreement.

Reply:
[Contains -> HasPrefix](https://github.com/Argus-Labs/world-engine/pull/626/commits/e35886b7bd703ab3c5375a118a78d22e59651f1f)
fixed